packages/nimble-blazor/Tests/NimbleBlazor.Tests/Unit/Components/NimbleAnchorButtonTests.cs

```diff
@@ -22,6 +23,15 @@ public void NimbleAnchorButton_Render_HasAnchorButtonMarkup()
         Assert.Contains(expectedMarkup, button.Markup);
     }
 
+    [Fact]
+    public void NimbleAnchorButton_SupportsAdditionalAttributes()
+    {
+        var context = new TestContext();
+        context.JSInterop.Mode = JSRuntimeMode.Loose;
```

Comment:
We can use Strict mode for these tests right? This shouldn't be invoking any JSRuntime behavior

Reply:
Well, some components might call interop code in something like `OnAfterRenderAsync` (the `Table` does this), in which case setting it to `Loose` is necessary.